### ISSUE TITLE
GridapMakie is no longer compatible with Makie 0.15 (and provably 0.13)

### ISSUE THREAD

**Issue by fverdugo** [gridap/GridapMakie.jl#62]
We need to change the compat section accordingly.

**Comment by hammy4815** [comment]
Hi @fverdugo , I've been able to locally remove the compat section (M1 chip), update all dependencies, and use the recipes without any issues. 

```
  [5789e2e9] FileIO v1.16.2
  [1a297f60] FillArrays v1.9.3
  [e9467ef8] GLMakie v0.9.6
  [5c1252a2] GeometryBasics v0.4.10
  [56d4f2e9] Gridap v0.17.23
  [ee78f7c6] Makie v0.20.5
```

Are there any issues that I overlooked? Otherwise, could we bump?

**Comment by hammy4815** [comment]
Hi @fverdugo ,

I've updated again to the most recent dependencies:

```
  [5789e2e9] FileIO v1.16.2
  [1a297f60] FillArrays v1.9.3
  [e9467ef8] GLMakie v0.9.8
  [5c1252a2] GeometryBasics v0.4.10
  [56d4f2e9] Gridap v0.17.23
  [ee78f7c6] Makie v0.20.7
```

And I still seem to be able to build plots using the recipes without encountering any problems.

**Comment by fverdugo** [comment]
Hi @hammy4815,

thanks for reporting.

can you modify the compat section (link below) with the new version numbers and then open a PR? 

https://github.com/gridap/GridapMakie.jl/blob/26399574aa6d5eb4c48c9e8079c04d02e9daf56a/Project.toml#L15

**Comment by hammy4815** [comment]
Sure, no problem @fverdugo . I just [opened a PR](https://github.com/gridap/GridapMakie.jl/pull/69#issue-2130601766).

**Comment by JordiManyer** [comment]
Closed by PR #73
